test/k8s/simple_test.go

```diff
@@ -73,7 +73,7 @@ data:
 			Expect(err).ToNot(HaveOccurred(), "invalid value of TEST_NUM_SERVICES")
 			numServices = i
 		}
-		services := graph.GenerateRandomServiceMesh(time.Now().Unix(), numServices, 50, 1, 1)
+		services := graph.GenerateRandomServiceMesh(GinkgoRandomSeed(), numServices, 50, 1, 1)
```

Comment:
I think we need to use constant number so the graph is actually the same on every test execution